Cargo.toml

```diff
@@ -3,7 +3,7 @@ name = "chan-signal"
 version = "0.3.0"  #:version
 authors = ["Andrew Gallant <jamslam@gmail.com>"]
 description = "Listen to OS signals with channels."
-documentation = "http://burntsushi.net/rustdoc/chan_signal/"
+documentation = "http://burntsushi.net/rustdoc/chan-signal/"
```

Comment:
Thanks! But actually, while you're here, could you just change this to `https://docs.rs/chan-signal`? Could you also update the link in the README? Thanks!